src/pheromones/hivemind-host/src/IdRequestDTO.cpp

```diff
@@ -0,0 +1,9 @@
+#include "IdRequestDTO.h"
+
+IdRequestDTO::IdRequestDTO(const IdRequest& req) { (void)req; }
+
+bool IdRequestDTO::serialize(IdRequest& req) const {
+    (void)req;
```

Comment:
Missing implementation?

Reply:
There is nothing to serialize (it has not fields), that's why there is not code